Packages/com.unity.inputsystem/InputSystem/Editor/Settings/InputSettingsProvider.cs

```diff
@@ -145,6 +145,17 @@ public override void OnGUI(string searchContext)
                 EditorGUILayout.Space();
                 EditorGUILayout.PropertyField(m_EditorInputBehaviorInPlayMode, m_EditorInputBehaviorInPlayModeContent);
 
+                EditorGUILayout.Space();
+                EditorGUILayout.LabelField("Shortcut support", EditorStyles.boldLabel);
```

Comment:
"Shortcut support" is referred to as "Improved Shortcut Support" below. Feel this should be consistent here. Also pascal case "support".

Reply:
Good catch